src/test/java/core/basesyntax/service/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static OperationHandler handler;
+    private static Operation operation;
+    private static Fruit fruit;
+    private FruitTransaction transaction;
+
+    @BeforeClass
+    public static void beforeClass() {
+        operation = Operation.BALANCE;
+        fruit = new Fruit("banana");
+        handler = new BalanceOperationHandler();
+        Storage.storage.clear();
+    }
+
+    @AfterClass
+    public static void afterClass() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    public void handle_Balance_Operation_ok() {
```

Comment:
```suggestion
    public void handle_ok() {
```
Follow the pattern: 
`<method name>_<description>_<expected behavior>`